nameparser/config/__init__.py

```diff
@@ -231,7 +231,7 @@ def __init__(self,
         self.first_name_titles   = SetManager(first_name_titles)
         self.conjunctions        = SetManager(conjunctions)
         self.capitalization_exceptions = TupleManager(capitalization_exceptions)
-        self.regexes             = TupleManager(regexes)
+        self.regexes             = TupleManager([tpl[:2] for tpl in REGEXES])
```

Comment:
I added a tag/label to some of the tuples.  The slice returns the first two items in the tuples, omitting the tag/label data.  The TupleManager object can still be used in the code.  The regexes variable in the constants() is no longer a set object, just a list of tuples.  I did this to preserve the order of the regex patterns.

Reply:
Ok. But If someone tries to instantiate passing a TuperManger to regexes, it will have no effect because you are using the global variable instead of the local one. Need to replace `REGEXES` for `regexes`. 

ex: `name = HumanNam(regexes=myTupleManager)` would fail. (I guess I should have some tests for those instantiation attributes.)

Reply:
I don't understand.  The list of compiled regex patterns for nicknames is different than the regexes that is fed into tuplemanager.  I thought I'd left the tuplemanager-based regexes alone.  I might have gotten a little confused by variables/functions with the same name.  I'll take another look at it.

Some clarification would be helpful.

Reply:
It's a simple mistake of using the module constant instead of the attribute passed to the class' init function.

change: 
`self.regexes             = TupleManager([tpl[:2] for tpl in REGEXES])`
to 
`self.regexes             = TupleManager([tpl[:2] for tpl in regexes])`

Here's a test that should pass but will fail with your code above.
```
    def test_custom_regex_constant(self):
        t = {'test': 'test'}
        c = Constants(regexes=t)
        self.assert_(c.regexes is t)
```

Reply:
(I edited the last line of my test to fix the equals test)